angular-workspace/projects/ni/nimble-angular/src/directives/table/nimble-table.directive.ts

```diff
@@ -7,6 +7,8 @@ import type { Observable, Subscription } from 'rxjs';
 export type { Table };
 export type { TableActionMenuToggleEventDetail, TableRowSelectionEventDetail };
 export { TableRecord, TableFieldName, TableFieldValue, TableValidity, TableRowSelectionMode };
+// eslint-disable-next-line no-restricted-imports
```

Comment:
A few thoughts:
1. Could we export this from a separate file with `testing` in its path? That way clients could set up their own `no-restricted-imports` rules to ban imports of testing code from production code. And instead of disabling `no-restricted-imports` here, we could change the [angular-workspace eslint configuration](https://github.com/ni/nimble/blob/874ed79d0d1c0f8228f72de3d12cf562e89d8ada/angular-workspace/.eslintrc.js#L37) to say that imports from testing paths are allowed if the path contains testing. (could probably just update the existing `*.spec.ts` pattern to include `*testing*` or something like that)
2. Once this is exported, can we remove [the suppression in the anchor column directive spec](https://github.com/ni/nimble/blob/874ed79d0d1c0f8228f72de3d12cf562e89d8ada/angular-workspace/projects/ni/nimble-angular/src/directives/table-column/anchor/tests/nimble-table-column-anchor-router-link-with-href.directive.spec.ts#L8)?

Reply:
I created entry points for the table, each column, and table testing. I believe everything you've outlined in your comment is addressed by that change (including updating the `testing` directories from being able to use `testing` code from nimble-components).